.eslint-doc-generatorrc.js

```diff
@@ -0,0 +1,11 @@
+/** @type {import('eslint-doc-generator').GenerateOptions} */
+const config = {
+  configEmoji: [
+    ['jsx-runtime', '🏃'],
+    ['recommended', '☑️'],
+  ],
+  ignoreConfig: ['all'],
+  urlConfigs: 'https://github.com/jsx-eslint/eslint-plugin-react/#shareable-configs',
+};
+
+module.exports = config;
```

Comment:
is there a reason this has to be a JS file? ideally config is static.

(it's also a bit confusing to have "rc" in the name when it's not actually an rc file, but i guess eslintrc already makes that mistake)

Reply:
Any filename or format from here is supported: https://github.com/davidtheclark/cosmiconfig#searchplaces

Note that only a JavaScript format enables us to benefit from the type definitions (auto-complete, hover-over documentation, etc).

Reply:
Since presumably the tool validates the config at runtime, I'm not super worried about the type definitions, but fair point.

Reply:
Yeah the types are just a convenience. The config file is fully validated at runtime.